oddbird/oddleventy
# Image brightness, and aria landmarks

![bioluminescence](https://www.hakaimagazine.com/wp-content/uploads/header-bioluminescence_0.jpg)

- [Trello card](https://trello.com/c/eOUyrp3T/35-i-want-images-to-be-less-intense-in-dark-mode-or-maybe-based-on-lightness-contrast) (partial)
- [Preview](https://deploy-preview-30--oddleventy.netlify.com/)

## Description
This branch went a bit beyond the original Trello card, because testing revealed a few other issues.

- I added the palette icon for the color menu
- Images now respond to a combination of light/dark mode, lightness, and contrast set in CCS.
- On hover/focus, they return to neutral state (`brightness(1) contrast(1)`)
- I was running into aria-landmark issues when I did accessibility testing, so I also re-arranged and labeled some landmarks on the site template. Not strictly color related…


## Steps to test/reproduce
- go to a page with a header image (see the first resource listed), and play with color mode/lightness/contrast settings
- navigate the site with a screen reader
